lib/command.js

```diff
@@ -141,10 +141,15 @@ class Command {
             debug('Finished updating directory');
         }
 
+        const system = new System(ui, extensions);
+
         if (!this.global) {
-            const checkValidInstall = require('./utils/check-valid-install');
+            const findValidInstall = require('./utils/find-valid-install');
 
-            checkValidInstall(commandName);
+            // CASE: directory was NOT specified (`-d /path/to/instance`)
+            // CASE: directory traversal is explicitly disabled
+            const shouldTraverse = !argv.dir && system.globalConfig.get('traverse_dirs', true);
```

Comment:
do we really need to make this a configurable setting? I guess there _could_ be privacy concerns, but I feel like people concerned about that level of privacy would also maybe chroot the user they run `ghost` commands with or change the read permissions to prevent the user from reading it.

Without a way to easily set this property (i.e. not opening the global config and editing it), I feel like adding in configurability isn't a huge need.

Reply:
It's been a while, but I think I added it because git supports something like that, but I agree, we could definitely get rid of thsi